go.mod

```diff
@@ -1,40 +1,59 @@
 module github.com/openshift/ocs-operator
 
-go 1.12
+go 1.13
 
 require (
 	cloud.google.com/go v0.40.0 // indirect
 	github.com/RHsyseng/operator-utils v0.0.0-20190807020041-5344a0f594b8
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/prometheus-operator v0.29.0
-	github.com/ghodss/yaml v1.0.0
+	github.com/coreos/prometheus-operator v0.34.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
-	github.com/go-openapi/spec v0.19.2
-	github.com/noobaa/noobaa-operator/v2 v2.0.6-0.20200219114615-8ae457e831e4
+	github.com/go-openapi/spec v0.19.4
+	github.com/noobaa/noobaa-operator/v2 v2.1.1
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
-	github.com/openshift/api v3.9.1-0.20190904155310-a25bb2adc83e+incompatible
-	github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
+	github.com/openshift/client-go v0.0.0-20190923180330-3b6373338c9b
 	github.com/openshift/custom-resource-status v0.0.0-20190812200727-7961da9a2eb7
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190605231540-b8a4faf68e36
-	github.com/operator-framework/operator-sdk v0.10.0
-	github.com/rook/rook v1.2.4
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20191115003340-16619cd27fa5
+	github.com/operator-framework/operator-sdk v0.16.0
+	github.com/rook/rook v1.3.2
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/zap v1.13.0
-	k8s.io/api v0.0.0-20191005115622-2e41325d9e4b
-	k8s.io/apiextensions-apiserver v0.0.0-20190820104113-47893d27d7f7
-	k8s.io/apimachinery v0.0.0-20191005115455-e71eb83a557c
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/cloud-provider v0.0.0-20191114015453-79225fba1e26 // indirect
-	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
-	sigs.k8s.io/controller-runtime v0.2.0
+	k8s.io/api v0.17.2
+	k8s.io/apiextensions-apiserver v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe
+	sigs.k8s.io/controller-runtime v0.5.1
 )
 
-// Pinned to deps for controller-runtime v0.2.0
+// Pinned to kubernetes-1.16.2 https://github.com/operator-framework/operator-sdk/blob/v0.16.0/internal/scaffold/go_mod.go
```

Comment:
I think you need `v0.17.2` for Rook-Ceph to work.

Reply:
trying with 0.17.2